app/Services/Account/Settings/ExportAccount.php

```diff
@@ -0,0 +1,1418 @@
+    }
+
+    /**
+     * Export the Term User table.
+     *
+     * @param array $data
+     */
+    private function exportTermUser(array $data)
+    {
+        $columns = [
+            'account_id',
+            'user_id',
+            'term_id',
+            'ip_address',
+            'created_at',
+            'updated_at',
+        ];
+
+        $foreignKey = 'account_id';
+
+        $this->buildInsertSQLQuery('term_user', $foreignKey, $columns, $data);
+    }
+
+    /**
+     * Export the User table.
+     *
+     * @param array $data
+     */
+    private function exportUser(array $data)
+    {
+        $columns = [
+            'id',
+            'first_name',
+            'last_name',
+            'email',
+            'me_contact_id',
+            'admin',
+            'email_verified_at',
+            'password',
+            'remember_token',
+            'google2fa_secret',
+            'account_id',
+            'timezone',
+            'currency_id',
+            'locale',
+            'metric',
+            'fluid_container',
+            'contacts_sort_order',
+            'name_order',
+            'invited_by_user_id',
+            'dashboard_active_tab',
+            'gifts_active_tab',
+            'profile_active_tab',
+            'profile_new_life_event_badge_seen',
+            'temperature_scale',
+            'created_at',
+            'updated_at',
+        ];
+
+        $foreignKey = 'account_id';
+
+        $this->buildInsertSQLQuery('users', $foreignKey, $columns, $data);
+    }
+
+    /**
+     * Export the Weather table.
+     *
+     * @param array $data
+     */
+    private function exportWeather(array $data)
+    {
+        $columns = [
+            'id',
+            'account_id',
+            'place_id',
+            'weather_json',
+            'created_at',
+            'updated_at',
+        ];
+
+        $foreignKey = 'account_id';
+
+        $this->buildInsertSQLQuery('weather', $foreignKey, $columns, $data);
+    }
+
+    /**
+     * Export the Contact Photo table.
+     * This is custom as we need to loop on the contacts for this account.
+     *
+     * @param array $data
+     */
+    private function exportContactPhoto(array $data)
+    {
+        $contacts = DB::table('contacts')
+            ->select('id')
+            ->where('account_id', $data['account_id'])
+            ->get();
+
+        if (! $contacts) {
+            throw new NoAccountException();
+        }
+
+        if ($contacts->count() == 0) {
+            return;
+        }
+
+        foreach ($contacts as $contact) {
+            $photos = DB::table('contact_photo')
+                ->where('contact_id', $contact->id)
+                ->get();
+
+            foreach ($photos as $photo) {
+                $sql = 'INSERT INTO contact_photo (contact_id, photo_id, created_at, updated_at) values (';
+                $sql .= $photo->contact_id.','.$photo->photo_id.",'".$photo->created_at."','".$photo->updated_at."');".PHP_EOL;
```

Comment:
Just need to finish this instructions. Why not using `$this->buildInsertSQLQuery` here?